cms/static/sass/elements/_modal-window.scss

```diff
@@ -667,6 +667,7 @@
         margin-left: 25px;
       }
 
+
```

Comment:
Seems like an unnecessary extra line -- plus it looks like the other classes have a preceding comment -- maybe include one here as well?